hw_7.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
Please fix a module name.

Reply:
done

Reply:
`Count lines file`?

It should be changed to `count_file_lines` or `calculate_lines_of_file`, etc.